src/services/analytics/events/assets.ts

```diff
@@ -17,19 +17,8 @@ export const ASSETS_EVENTS = {
     action: 'Tokens',
     category: ASSETS_CATEGORY,
   },
-  // TODO: How should we now track this as we paginate items?
-  NFT_AMOUNT: {
-    event: EventType.META,
-    action: 'NFTs',
-    category: ASSETS_CATEGORY,
-  },
```

Comment:
We now paginate NFTs and therefore can't track the amount. The removal was confirmed by @johannesmoormann [here](https://5afe.slack.com/archives/C03DAGWJCR5/p1663329731217239?thread_ts=1663328560.379069&cid=C03DAGWJCR5).

Reply:
I also feel like this should be tracked through dune analytics rather than the front-end.